### ISSUE TITLE
[ Sugestion ] Add playing status (spotify, minecraft, anything)

### ISSUE THREAD

**Issue by skhrlx** [Vedza/NitroSniperGo#350]
By this way, the account can get more trust if we won a giveaway and the people think what we are a bot.

Add too a auto reset, like, "playing minecraft at 48 hours", no one play a game 48hours per day, so, when the bot played minecraft for 12 hours, the bot resets, and start a new count playing status

{Sry. For the bad english, and minecraft it's just a example, thank's!}

**Comment by raVenoxis** [comment]
just run a discord rpc on heroku

**Comment by apex992** [comment]
> just run a discord rpc on heroku

hey sorry to bother you, is there any tutorial or any repo, im newbie. thanks

**Comment by slowwdev** [comment]
> 

I made a discord.js selfbot to have discord rpc like spotify, twitch or a simple game.

I will make a repo and a youtube tutorial soon.
Add me on discord: 14ت#0002

I know my nickname is bugged lmfao but copy and past in friends addding tab.

**Comment by slowwdev** [comment]
> > just run a discord rpc on heroku
> 
> hey sorry to bother you, is there any tutorial or any repo, im newbie. thanks

Dm me ( i know my nickname is bugged)
```
@14ت#0002 
```

**Comment by Vedza** [comment]
> > just run a discord rpc on heroku
> 
> hey sorry to bother you, is there any tutorial or any repo, im newbie. thanks

Read that https://github.com/Vedza/NitroSniperGo#usage

It takes a register and 4 clicks.

And if you added the guy on discord, be careful, you should not run anything he is sending unless it's a link from this repository.

**Comment by slowwdev** [comment]
> > > just run a discord rpc on heroku
> > 
> > 
> > hey sorry to bother you, is there any tutorial or any repo, im newbie. thanks
> 
> Read that https://github.com/Vedza/NitroSniperGo#usage
> 
> It takes a register and 4 clicks.
> 
> And if you added the guy on discord, be careful, you should not run anything he is sending unless it's a link from this repository.

mdr vedza t'es srx on se connait tu sais que je fais pas ça en plus j ai vraiment dev un selfbot rpc c est pas un token grabber

**Comment by slowwdev** [comment]
> > just run a discord rpc on heroku
> 
> hey sorry to bother you, is there any tutorial or any repo, im newbie. thanks

I will create a repo for you man

**Comment by slowwdev** [comment]
> > just run a discord rpc on heroku
> 
> hey sorry to bother you, is there any tutorial or any repo, im newbie. thanks

i chatted with vedza on discord and he authorized me to made my repo public and post the link here.

so just wait a little bit because i need to create a readme for my repo.
then i will paste the link in this issue.

**Comment by apex992** [comment]
> > > just run a discord rpc on heroku
> > 
> > 
> > hey sorry to bother you, is there any tutorial or any repo, im newbie. thanks
> 
> i chatted with vedza on discord and he authorized me to made my repo public and post the link here.
> 
> so just wait a little bit because i need to create a readme for my repo. then i will paste the link in this issue.

oh okay thanks

**Comment by apex992** [comment]
hey there it has been 5 days are you still working on it?

**Comment by slowwdev** [comment]
> hey there it has been 5 days are you still working on it?
3 days* 
Just wait uh 2 days i am making a tutorial for how to host for free on replit on my readme.

**Comment by slowwdev** [comment]
> > hey there it has been 5 days are you still working on it?
> 
> 3 days* Just wait uh 3 days i am making a tutorial for how to host for free on replit on my readme.
and im not a pro dev lol i need to play video games sometimes

**Comment by apex992** [comment]
> > hey there it has been 5 days are you still working on it?
> 
> 3 days* Just wait uh 3 days i am making a tutorial for how to host for free on replit on my readme. and i need to play video games sometimes lol

oh ok

**Comment by slowwdev** [comment]
https://github.com/slowwdev/Discord-Selfbot-RPC I will post tutorial in wiki soon !

**Comment by apex992** [comment]
> https://github.com/slowwdev/Discord-Selfbot-RPC I will post tutorial in wiki soon ! @apex992

ayyy thanks man!

**Comment by slowwdev** [comment]
> > https://github.com/slowwdev/Discord-Selfbot-RPC I will post tutorial in wiki soon ! @apex992
> 
> ayyy thanks man! it can be deployed on heroku?

Yes but you cant deploy 2 apps on heroku with the free plan (or your apps will stop after many days).
But if you really want to deploy on heroku i can do an heroku version with a tutorial :)

**Comment by apex992** [comment]
> > > https://github.com/slowwdev/Discord-Selfbot-RPC I will post tutorial in wiki soon ! @apex992
> > 
> > 
> > ayyy thanks man! it can be deployed on heroku?
> 
> Yes but you cant deploy 2 apps on heroku with the free plan (or your apps will stop after many days). But if you really want to deploy on heroku i can do an heroku version with a tutorial :)

hey sorry for replying late but it would be great if there would be heroku deployment thanks!

**Comment by slowwdev** [comment]
> > > > https://github.com/slowwdev/Discord-Selfbot-RPC I will post tutorial in wiki soon ! @apex992
> > > 
> > > 
> > > ayyy thanks man! it can be deployed on heroku?
> > 
> > 
> > Yes but you cant deploy 2 apps on heroku with the free plan (or your apps will stop after many days). But if you really want to deploy on heroku i can do an heroku version with a tutorial :)
> 
> hey sorry for replying late but it would be great if there would be heroku deployment thanks!

ok i willl do
